src/main/docker/Dockerfile.noPlugins

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright 2022-2024 Open Text.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Docker registry
+ARG DOCKER_HUB_PUBLIC=docker.io
+
+# Reference the official OpenSearch 1 image
+FROM @DOCKER_HUB_PUBLIC@/opensearchproject/opensearch:1.3.15 AS os1-image
+
+#
+# The actual image definition
+#
+FROM @DOCKER_HUB_PUBLIC@/cafapi/prereleases:opensuse-opensearch1-2.8.0-SNAPSHOT
+
+# Copy jdk from OpenSearch 1 image
+COPY --from=os1-image /usr/share/opensearch/jdk /usr/share/opensearch/jdk
```

Comment:
COPY --from=os1-image /usr/share/opensearch/jdk /usr/share/opensearch/jdk

This will copy the jdk into the history of the image